foodsaving/invitations/serializers.py

```diff
@@ -17,7 +17,9 @@ class Meta:
             UniqueTogetherValidator(
                 queryset=Invitation.objects.filter(expires_at__gte=timezone.now()),
                 fields=('email', 'group'),
-                message=_('An invitation has already been sent to this e-mail address')
+                message=_(
+                    'An invitation has already been sent to this e-mail address!'
```

Comment:
Maybe remove the added newlines, then the diff is clean and nice :)

Reply:
I forgot to reverse that change, after changing the message, sorry about that.